app/index.html

```diff
@@ -5,8 +5,10 @@
     <link rel="stylesheet"
       href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.12/semantic.min.css">
     </link>
+    <!--<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">-->
```

Comment:
This commented out line should be removed.

Reply:
Please approve this part as of now. I am not making a commit for such a trivial thing. Will update this on the next pull request.

Reply:
@sagnikm95 You may also amend your previous commit. No need to make a new commit to make trivial changes. That way it will be cleaner.